admins/pageflow/user.rb

```diff
@@ -41,7 +41,9 @@ module Pageflow
           row :locale do
             I18n.t('language', locale: user.locale)
           end
-          boolean_status_tag_row(:admin?, 'admin warning')
+          if authorized?(:see_admin_status_of, user)
+            boolean_status_tag_row(:admin?, 'admin warning')
+          end
         end
```

Comment:
Let's add a [controller spec](https://github.com/aviav/pageflow/blob/007e4a08566642a6131a6996d2ba0ad326446997/spec/controllers/admin/users_controller_spec.rb#L5), that makes sure this code continues to work.